strace.py

```diff
@@ -280,4 +282,4 @@ def createChild(self, program):
 
 
 if __name__ == "__main__":
-    exit(SyscallTracer().main())
+    sys.exit(SyscallTracer().main())
```

Comment:
https://docs.python.org/3/library/os.html#os._exit

Reply:
I would prefer to call sys.exit() in the main() method, at main() exit.

Reply:
Thanks, I applied all suggestions.